### PR TITLE
perf: faster test run

### PR DESCRIPTION
**Pull request by ronag**
Makes `npm run test` more than twice as fast.

### CODE REVIEW

**Comment by ronag** [comment]
> if you disable typescript, flow, esm, and jsx you can achieve the same speed without removing functionality.

Changed. It's faster... but not as far. 22s vs 14s

**Comment by mcollina** [comment]
running code coverage for every run locally is important, otherwise prs will fail while it would pass locally.

**Comment by ronag** [comment]
> running code coverage for every run locally is important, otherwise prs will fail while it would pass locally.

I'm original suggestion was to always run it through the pre-commit hook, but not when manually running `npm test`.

**Comment by mcollina** [comment]
Ok, let's do that.

**Comment by ronag** [comment]
PTAL

File: package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "description": "An HTTP/1.1 client, written from scratch for Node.js",
   "main": "index.js",
   "scripts": {
-    "test": "standard | snazzy && tap test/*.js --100 --no-ts --no-jsx --no-flow --no-esm"
+    "lint": "standard | snazzy",
+    "test": "tap test/*.js --no-coverage",
+    "coverage": "standard | snazzy && tap test/*.js --100 --no-ts --no-jsx --no-flow --no-esm"
```
Comment (mcollina) [inline package.json:9]: please move the `--no-ts --to-jsx --no-flow --no-esm` to a `.taprc` file. See https://github.com/fastify/fastify/blob/master/.taprc

Comment (ronag) [reply package.json]: I guess coverage: true is the same as --100?

**Pull request merged**